tests/acceptance/PaymentCept.php

```diff
@@ -71,12 +68,12 @@
 
 $I->amGoingTo('send third payment');
 $I->click('//a[contains(@class, \'ui--sendEmail\')]');
-$I->waitForElement('//*[@title="Odeslané emaily"]');
+//$I->waitForElement('//*[@title="Odeslané emaily"]');
```

Comment:
Tohle je zakomentované proč?

Reply:
kdyz tam není ten mailSmtpCatcher, tak to není jak kontrolovat a zatím jsem to jen komentoval, ale odeberu to uplne

Reply:
Můžeš mi to pls dovysvětlit, tohle nepracuje s MailCatcherem, ale testuje, že v UI je u té platby následně ikonka, že je platba odeslána.
![image](https://user-images.githubusercontent.com/5658260/96441049-259d3f00-1209-11eb-876a-a578201e00f9.png)

Reply:
ok, to je asi omyl, uz si nepamatuju proc jsem to delal